src/main/java/com/hivemq/cli/commands/AbstractCommand.java

```diff
@@ -13,33 +13,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.hivemq.cli.commands;
 
 import picocli.CommandLine;
 
-@CommandLine.Command(sortOptions = false,
-        synopsisHeading = "%n@|bold Usage|@:  ",
-        synopsisSubcommandLabel = "{ pub | sub | shell }",
-        descriptionHeading = "%n",
-        optionListHeading = "%n@|bold Options|@:%n",
-        commandListHeading = "%n@|boldCommands|@:%n",
-        separator = " ")
+@CommandLine.Command(sortOptions = false, synopsisHeading = "%n@|bold Usage|@:  ",
```

Comment:
:(

Reply:
This was a lot more readable before

Reply:
Will be addressed with the future editorconfig. Except you want me to add formatter comments.